### DISCUSSION TITLE
Can you update fasterxml.jackson to 2.14.2 (or later)?

### DISCUSSION THREAD

**Discussion by pcreager23** [dropwizard/dropwizard#6865] [General]
@joschi, et al,
I know that dropwizard is not in fact affected by the https://github.com/advisories/GHSA-mjmj-j48q-9wg2 bug in snakeyaml, however several scanners still flag dropwizard since it contains the pre-patched version of snakeyaml, by way of Jackson/FasterXML. (transitive dependency)

So could you please upgrade fasterxml.jackson to 2.14.2 (or later) to quiet these scanners?

Thank you in advance from those of us in the trenches in Corporate IT!

**Comment by joschi** [comment]
@pcreager23 We won't update to Jackson 2.14.x in the Dropwizard 2.1.x branch.

Even though there are no (known?) breaking changes, we want to avoid bad surprises for our users.

This being said, I think you have two ways forward:

- Upgrade to Dropwizard 3.0.0-beta.5 which will be released as Dropwizard 3.0.0 GA soon. There we are already referencing Jackson 2.14.2: https://github.com/dropwizard/dropwizard/blob/v3.0.0-beta.5/dropwizard-dependencies/pom.xml#L39
- Update Jackson (via [Jackson BOM](https://github.com/FasterXML/jackson-bom)) to version 2.14.2 in your application. The security scanners usually scan the classpath of your project and not the unresolved dependencies.

**Reply by pcreager23** [reply]
Excellent, thank you.
